app/views/admin/pages/edit.html.erb

```diff
@@ -19,13 +19,13 @@
   </p>
   <p>
     Template:
-    <br><%= select( 'page', 'template_id',
-      PageTemplate.all.collect { |t| [ t.name, t.id ] } ) %>
+    <br><%= select( 'page', 'template_id', PageTemplate.pluck( :name, :id ),
```

Comment:
Consider using the form builder (`f`) `select` method, and it ought to set the `selected` option correctly:

```erb
<%= f.select(:template_id,  PageTemplate.pluck( :name, :id ) %>
```

Reply:
Aha, cool, _much_ neater.

What you see here is what I found when googling for syntax, I did wonder why it wasn't an f.* like the rest.